classes/class-dintero-checkout-callback.php

```diff
@@ -92,51 +98,58 @@ public function callback() {
+					Dintero_Checkout_Logger::log( sprintf( 'CALLBACK [%s]: The status for the WC order id %s / %s (transaction id: %s) was changed to CAPTURE in the back office.', $event, $order_id, $merchant_reference, $transaction_id ) );
 					if ( ! Dintero()->order_management->is_captured( $order_id, true ) ) {
 						$order->add_order_note( __( 'The order was CAPTURED in the Dintero backoffice.', 'dintero-checkout-for-woocommerce' ) );
 					}
 					break;
 
 				case 'REFUND':
-					Dintero_Logger::log( sprintf( 'CALLBACK [%s]: The status for the WC order id %s / %s (transaction id: %s) was changed to REFUND (or PARTIALLY REFUNDED) in the back office.', $event, $order_id, $merchant_reference, $transaction_id ) );
+					Dintero_Checkout_Logger::log( sprintf( 'CALLBACK [%s]: The status for the WC order id %s / %s (transaction id: %s) was changed to REFUND (or PARTIALLY REFUNDED) in the back office.', $event, $order_id, $merchant_reference, $transaction_id ) );
 					if ( ! Dintero()->order_management->is_refunded( $order_id, true ) ) {
 						$order->add_order_note( __( 'The order was REFUNDED in the Dintero backoffice.', 'dintero-checkout-for-woocommerce' ) );
 					}
 					break;
 
 				case 'VOID':
-					Dintero_Logger::log( sprintf( 'CALLBACK [%s]: The status for the WC order id %s / %s (transaction id: %s) was changed to VOID in the back office.', $event, $order_id, $merchant_reference, $transaction_id ) );
+					Dintero_Checkout_Logger::log( sprintf( 'CALLBACK [%s]: The status for the WC order id %s / %s (transaction id: %s) was changed to VOID in the back office.', $event, $order_id, $merchant_reference, $transaction_id ) );
 					if ( ! Dintero()->order_management->is_canceled( $order_id, false ) ) {
 						$order->update_status( 'cancelled', __( 'The order was CANCELED in the Dintero backoffice.', 'dintero-checkout-for-woocommerce' ) );
 					}
 					break;
 
 				default:
-					Dintero_Logger::log( sprintf( 'CALLBACK [%s] unknown, ignored for WC order id: %s / %s (transaction id: %s). ' . json_encode( filter_var_array( $_GET, FILTER_SANITIZE_STRING ) ), $event, $order_id, $merchant_reference, $transaction_id ) );
+					Dintero_Checkout_Logger::log( sprintf( 'CALLBACK [%s] unknown, ignored for WC order id: %s / %s (transaction id: %s). ' . json_encode( filter_var_array( $_GET, FILTER_SANITIZE_STRING ) ), $event, $order_id, $merchant_reference, $transaction_id ) );
 					break;
 			}
 
-			return;
+			header( 'HTTP/1.1 200 OK' );
+			die;
 		}
 
 		// Check if the order is set to on-hold, awaiting authorization.
 		$dintero_order = Dintero()->api->get_order( $transaction_id );
-		$is_authorized = ( 'AUTHORIZED' === $dintero_order['result']['status'] );
+		if ( is_wp_error( $dintero_order ) ) {
+			header( 'HTTP/1.1 500 Internal Server Error' );
```

Comment:
Should there be logging here, or is it handled in `get_order`?

Reply:
It is handled in `get_order`.